src/main/kotlin/uk/gov/justice/digital/hmpps/visitscheduler/service/SessionService.kt

```diff
@@ -59,10 +59,13 @@ class SessionService(
       prisonerValidationService.validatePrisonerIsFromPrison(prisonerId, prisonCode)
     }
 
+    val inclEnhancedPrivilegeTemplates = prisonerId?.let { prisonApiService.hasPrisonerGotEnhancedPrivilege(prisonerId) } ?: run { true }
```

Comment:
nice 1 line implementation. However should it default to false if  prisonerId is null or am i reading this wrong :)

Reply:
Yeha this is wired one, basically I'm saying if we not searching by prisonerId then return all session templates.